groups/prometheus/variables.tf

```diff
@@ -51,6 +81,15 @@ variable "service" {
   description = "The service name to be used when creating AWS resources"
 }
 
+variable "vault_username" {
+  type        = string
+  description = "The username used by the Vault provider."
```

Comment:
I'd go with something like:
`description = "Username for connecting to Vault - usually supplied through TF_VARS" `

Reply:
Description updated.